src/App.css

```diff
@@ -1,39 +1,299 @@
+  justify-content: space-between;
+  align-items: center;
+}
+
+.bookInfo {
+  display: flex;
+  flex-direction: column;
+}
+
+.catTitAuth {
+  display: flex;
+  flex-direction: column;
+}
+
+.catTitAuth p {
+  margin: 0 39px 0 0;
+  opacity: 0.5;
+  font-family: var(--monsterat);
+  font-size: 14px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+.catTitAuth h4 {
+  font-family: var(--robotoSlab);
+  font-size: 14px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--cool-blue);
+}
+
+.catTitAuth h2 {
+  font-family: var(--robotoSlab);
+  font-size: 22px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+}
+
+.bookBtns {
+  display: flex;
+  margin-top: 20px;
+}
+
+.bookBtns button {
+  background: none;
+  border: none;
+  font-family: var(--robotoSlab);
+  font-size: 14px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--cool-blue);
+  cursor: pointer;
+}
+
+.bookBtns button:first-child {
+  padding-right: 20px;
+  padding-left: 0;
+}
+
+.bookBtns button:not(:first-child) {
+  padding: 0 20px;
+}
+
+.bookBtns button:not(:last-child) {
+  border-right: solid 1px #e8e8e8;
+}
+
+.circleContainer {
+  width: 68px;
+  height: 68px;
+  border-style: solid;
+  border-width: 5px;
+  border-image-source: linear-gradient(to bottom, var(--medium-blue), var(--dodger-blue));
+  border-image-slice: 1;
+}
+
+.addbookform {
+  border-top: solid 1px #e8e8e8;
+  display: flex;
+  flex-direction: column;
+  width: 90vw;
+  margin: 20px auto;
+  justify-items: center;
+  padding-top: 20px;
+}
+
+.addbookform h2 {
+  color: var(--warm-grey);
+  margin-bottom: 20px;
+}
+
+.form {
+  display: flex;
+}
+
+.submit_button {
+  width: 184px;
+  text-transform: uppercase;
   color: white;
+  border: none;
+  border-radius: 3px;
+  background-color: var(--azure);
+  margin-left: auto;
 }
 
-.App-link {
-  color: #61dafb;
+.input_box {
+  width: 60%;
+  height: 40px;
+  opacity: 0.2;
+  padding: 10px;
+  background-color: var(--white-two);
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+.input_box:focus {
+  opacity: 1;
+}
 
-  to {
-    transform: rotate(360deg);
-  }
+.category {
+  flex: 0 1 250px;
+  margin-left: 50px;
+  opacity: 0.2;
+}
+
+.category:focus {
+  opacity: 1;
+}
+
+.percentage {
+  margin-left: 20px;
+}
+
+.percentage h2 {
+  font-family: var(--monsterat);
+  font-size: 32px;
+  color: var(--black-two);
+}
+
+.percentage p {
+  opacity: 0.5;
+  font-family: var(--monsterat);
+  font-size: 14px;
+  color: var(--black-two);
+}
+
+.progress {
+  display: flex;
+  align-items: center;
+}
+
+.lastElement {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
## Aim to Stay `DRY`
- [ ] There are some repeated rules in line `#262` & `#267`. Do you mind merging these rules as they appear similar so as to not appear like you are repeating yourself?